.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+languaje: node_js
```

Comment:
Travis validate this file and warns us that it does not recognize "languaje", instead we must use "language"

Reply:
all changes done